app/mixins/event-wizard.js

```diff
@@ -59,59 +59,76 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
         }
       }
     }
+    const numberOfTickets = data.tickets.length;
```

Comment:
This code will completely crash if for some reason data.tickets is not defined.
Every where else, first it is checked if data.property exists. Data.undefined.length will throw an exception.

Reply:
@CosmicCoder96 Understood, changing accordingly

Reply:
@CosmicCoder96 Updated the assignment, please check now